eng/Version.Details.xml

```diff
@@ -174,9 +174,9 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>ee0a97a0009c0e048789126253fea7994db676ac</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21616.4" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21519.2" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/source-build</Uri>
-      <Sha>90bdf447e1b97605f109b34243ab8c9f215308e9</Sha>
+      <Sha>10d0f7e94aa45889155c312f51cfc01bf326b853</Sha>
```

Comment:
This is weird - this CPD _is_ bringing us up-to-date with the version in dotnet/sdk, but that version was last updated 5 months ago - why wasn't the CPD updated before that? @dotnet/dnceng @mmitche 

https://github.com/dotnet/sdk/blob/0d872e73ea6586fe42ebdb507cd9d99bf9cf5571/eng/Version.Details.xml#L280

Reply:
Same is happening in this PR: https://github.com/dotnet/installer/pull/13409/files

Reply:
same here https://dev.azure.com/dnceng/internal/_git/dotnet-installer/pullrequest/21732

Reply:
Yep, it looks like a legacy coherency issue. I added this dependency here: https://dev.azure.com/dnceng/internal/_git/dotnet-installer/commit/342ef729e20a65bcc50a2396aa88e346b4a4e39e?refName=refs%2Fheads%2Fdarc-internal%2Frelease%2F6.0.2xx-e04054ba-ab03-4db2-b352-bc2c4de2e743,

That was needed to make that dep correct, but broke coherency because runtime doesn't know about the EMSDK band dependency. When coherency was updated, it downgraded.  I think we should take this change, then get SDK on the right version.